include/odp/api/spec/packet.h

```diff
@@ -50,29 +50,24 @@ extern "C" {
  * Invalid packet segment
  */
 
- /**
-  * @typedef odp_packet_color_t
-  * Color of packet for shaper/drop processing
-  */
-
- /**
-  * @def ODP_PACKET_GREEN
-  * Packet is green
-  */
-
- /**
-  * @def ODP_PACKET_YELLOW
-  * Packet is yellow
-  */
-
- /**
-  * @def ODP_PACKET_RED
-  * Packet is red
-  */
+/**
+ * @enum odp_packet_color_t
+ * Color of packet for shaper/drop processing
+ *
+ * @var ODP_PACKET_GREEN
+ * Packet is green
+ *
+ * @var ODP_PACKET_YELLOW
+ * Packet is yellow
+ *
+ * @var ODP_PACKET_RED
+ * Packet is red
+ */
 
 /**
- * Maximum number of packet colors which accommodates ODP_PACKET_GREEN, ODP_PACKET_YELLOW and
- * ODP_PACKET_RED.
+ * @def ODP_NUM_PACKET_COLORS
+ * Maximum number of packet colors which accommodates ODP_PACKET_GREEN,
+ * ODP_PACKET_YELLOW and ODP_PACKET_RED
```

Comment:
This change is not needed since the actual define is in this file. You can add my and Petri's reviewed-by tags on the same go.